Controller/SettingsController.php

```diff
@@ -24,7 +24,10 @@ public function modifyAction(Request $request) {
 			'settings' => $allStoredSettings,
 		);
 
-		$form = $this->createForm(new ModifySettingsForm(), $formData);
+		$form = Kernel::VERSION_ID < 20800
+			? $this->createForm(new LegacyModifySettingsForm(), $formData)
+			: $this->get('form.factory')->createNamed('craue_config_modifySettings', 'Craue\ConfigBundle\Form\ModifySettingsForm', $formData);
```

Comment:
I never thought about using one version of the Form component in combination with a different version of the framework itself. Is that even reasonable? And if so, I'd also need to test such cases, which in turn would horribly increase the test matrix for Travis. I'd prefer a way to ensure consistent versions being used.